src/propagation.jl

```diff
@@ -214,16 +189,16 @@ for V_dense in (:(Val{true}), :(Val{false}))
             tmax = t0 + tspan*yr
 
             # Integration
-            sol_ = @time taylorinteg_threads(dynamics, q0, t0, tmax, order, abstol, $V_dense(), params, maxsteps = maxsteps,
-                                             parse_eqs = parse_eqs)
+            sol = @time taylorinteg(dynamics, q0, t0, tmax, order, abstol, $V_dense(), params, 
+                                     maxsteps = maxsteps, parse_eqs = parse_eqs)
```

Comment:
I'd suggest to simplify this a bit as:
```suggestion
            sol = @time taylorinteg(dynamics, q0, t0, tmax, order, abstol, $V_dense(), params; 
                                     maxsteps, parse_eqs)
```